catalog/view/theme/default/template/extension/payment/securesubmit.tpl

```diff
@@ -1,156 +1,327 @@
 <link rel="stylesheet" type="text/css" href="catalog/view/stylesheet/securesubmit.css">
-<form class="form-horizontal">
-    <fieldset id="payment">
-        <legend><?php echo $text_credit_card; ?><br>
 
-            <div class="ss-shield">  </div>
-            <div class="visa-gray hidden-xs "></div>
-            <div class="mc-gray hidden-xs"></div>
-            <div class="amex-gray hidden-xs"></div>
-            <div class="jcb-gray hidden-xs"></div>
-            <div class="disc-gray hidden-xs"></div>
+<!-- make iframes styled like other form -->
+<style type="text/css">
+    #iframes iframe{
+        float:left;
+        width:100%;
+    }
+    .iframeholder:after,
+    .iframeholder::after{
+        content:'';
+        display:block;
+        width:100%;
+        height:0px;
+        clear:both;
+        position:relative;
+    }
+</style>
 
-        </legend>
 
-        <!--  <div class="form-group required col-md-10 ">
-
-            <label class="control-label ss-label" for="input-cc-owner"><?php echo $entry_cc_owner; ?></label></br>
-
-              <input type="text" name="cc_owner" value="" placeholder="<?php echo $entry_cc_owner; ?>" id="input-cc-owner" class="form-control ss-form-control" />
-
-          </div> -->
-        <div class="form-group required col-md-10">
-
-            <label class="control-label ss-label" for="input-cc-number"><?php echo $entry_cc_number; ?></label></br>
-
-            <input type="text" value="" placeholder="•••• •••• •••• ••••" id="input-cc-number" class="form-control ss-form-control card-type-icon" />
-
-        </div>
-
-        <div class="form-group required col-md-5">
-            <label class="control-label ss-label" for="input-cc-expire-date"><?php echo $entry_cc_expire_date; ?></label></br>
-
-            <input type="text" name="cc_expire_date" id="input-cc-expire-date" class="form-control ss-form-control" placeholder="MM / YYYY" />
-
-        </div>
-        <div class="form-group required col-md-5 col-md-offset-7">
-            <label class="control-label ss-label cvv-label" for="input-cc-cvv2"><?php echo $entry_cc_cvv2; ?></label></br>
-
-            <input type="text" value="" placeholder="<?php echo $entry_cc_cvv2; ?>" id="input-cc-cvv2" class="form-control ss-form-control cvv-icon"  />
+<!-- The Payment Form -->
+<fieldset id="payment">
+    <legend><?php echo $text_credit_card; ?><br>
+        <div class="ss-shield">  </div>
+        <div class="visa-gray hidden-xs "></div>
+        <div class="mc-gray hidden-xs"></div>
+        <div class="amex-gray hidden-xs"></div>
+        <div class="jcb-gray hidden-xs"></div>
+        <div class="disc-gray hidden-xs"></div>
+    </legend>
+</fieldset>
+<form id="iframes" action="" method="GET">
```

Comment:
This form element isn't required and should be removed.

Reply:
the form element removed